TextMining.py

```diff
@@ -0,0 +1,84 @@
+import requests
+import pickle
+import string
+import random
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+#data file downloaded once, text has been saved through pickle
```

Comment:
You can use ` """ comments """ ` instead of putting `#` for every line of code. This is called the header comment. It usually contains brief description of the code, just like what you've written here.